docs/opa-entity-checker/quick-start.md

```diff
@@ -187,6 +187,27 @@ opaClient:
       entrypoint: 'entity_checker/violation'
 ```
 
+## Adding the Entity Processor
+
+Run the following command to install the OPA Entity Checker Entity Processor in your Backstage project.
+
+```bash
+yarn add --cwd packages/backend @parsifal-m/backstage-plugin-catalog-backend-module-entity-checker-processor
```

Comment:
If we decide to rename it, we'd need to rename it here to `backstage-plugin-opa-entity-checker-processor`